src/main/java/frc/robot/OI.java

```diff
@@ -55,6 +58,8 @@ private OI() {
             b.whenPressed(new GotoPosition(e.getValue()));
             this.boardButtons[e.getKey()] = b;
         }
+        winchClimber = new JoystickButton(bb, RobotMap.OI.WINCH_CLIMBER);
```

Comment:
When you add a button from the button board, you should add it to the `boardButtons` array at the same index as the button. (So you'd add `boardButtons[RobotMap.OI.WINCH_CLIMBER] = winchClimber;` here.)

Reply:
To be 100% honest, this isn't incredibly important, it's mostly for consistency with the arm go-to-position buttons being in that array.